ray-operator/test/sampleyaml/support.go

```diff
@@ -101,3 +103,24 @@ func SubmitJobsToAllPods(t Test, rayCluster *rayv1.RayCluster) func(Gomega) {
 		}
 	}
 }
+
+func QueryDashboardGetAppStatus(t Test, rayCluster *rayv1.RayCluster) func(Gomega) {
+	return func(g Gomega) {
+		rayDashboardClient := &utils.RayDashboardClient{}
+		pod, err := GetHeadPod(t, rayCluster)
+		g.Expect(err).ToNot(HaveOccurred())
+
+		localPort := 8265
+		remotePort := 8265
+		stopChan, err := SetupPortForward(t, pod.Name, pod.Namespace, localPort, remotePort)
+		defer close(stopChan)
+
+		g.Expect(err).ToNot(HaveOccurred())
+		url := fmt.Sprintf("127.0.0.1:%d", localPort)
+
+		err = rayDashboardClient.InitClient(t.Ctx(), url, rayCluster)
+		g.Expect(err).ToNot(HaveOccurred())
+		_, err = rayDashboardClient.GetServeDetails(t.Ctx())
```

Comment:
Can you also check whether all Serve application statuses on the Ray dashboard are `rayv1.ApplicationStatusEnum.RUNNING`?

Reply:
@kevin85421 I think this has been addressed in https://github.com/ray-project/kuberay/pull/2496 right?

Reply:
They are different. #2496 checks RayService status, and this PR checks Ray dashboard which is the source of the truth.

Reply:
OK, No problem I will handle that.